charts/datadog/templates/agent-cilium-network-policy.yaml

```diff
@@ -108,6 +115,52 @@ specs:
           - ports:
               - port: "10250"
                 protocol: TCP
+              - port: "6942"
+                protocol: "TCP"
```

Comment:
What is the `6942` port about ?
Whereas I could find the `10250` as being the default kubelet port on [this page](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/), I couldn’t find any mention of `6942`.